pkg/plugins/trigger/pull-request.go

```diff
@@ -242,23 +250,66 @@ I understand the commands that are listed [here](https://jenkins-x.io/v3/develop
 	return nil
 }
 
-// TrustedPullRequest returns whether or not the given PR should be tested.
-// It first checks if the author is in the org, then looks for "ok-to-test" label.
-func TrustedPullRequest(spc scmProviderClient, trigger *plugins.Trigger, author, org, repo string, num int, l []*scm.Label) ([]*scm.Label, bool, error) {
-	// First check if the author is a member of the org.
-	if orgMember, err := TrustedUser(spc, trigger, author, org, repo); err != nil {
-		return l, false, fmt.Errorf("error checking %s for trust: %v", author, err)
-	} else if orgMember {
-		return l, true, nil
+func welcomeMsgForDraftPR(spc scmProviderClient, trigger *plugins.Trigger, pr scm.PullRequest) error {
+	org, repo, _ := orgRepoAuthor(pr)
+
+	comment := "This is a draft PR with trigger.SkipDraftPR enabled on this repository\n"
```

Comment:
Is this for debugging, should we update this comment? Feels quite techy for a developer that maybe isn't familiar with jenkins x. We could always link docs alongside a more non jenkins x'y comment. or can we add a label instead similar to the needs-ok-to-test

Reply:
![Screenshot_20240205_134636_Chrome](https://github.com/jenkins-x/lighthouse/assets/45177808/a415ccb1-a986-42d3-ac72-fd39f53251d7)

^ This would be the complete message for a Trusted user.
If the user is untrusted, then the message would be a little bit different.

Did you comment on the first part "This is a draft PR with trigger.SkipDraftPR enabled on this repository\n" or the whole message?

Reply:
That's fine, if we can fix the spacing of the message then I'm happy to merge :)

Reply:
Ok it's the new proposed version :+1: 

![Screenshot from 2024-02-06 16-13-42](https://github.com/jenkins-x/lighthouse/assets/45177808/e7262ae6-5e7a-4373-8709-18a0a3f1a1f9)


![Screenshot from 2024-02-06 16-13-52](https://github.com/jenkins-x/lighthouse/assets/45177808/75a6469a-9419-428e-802c-300b97f9c28f)